src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write code here
+const li = document.querySelectorAll('li');
```

Comment:
It's recommended to use a more descriptive variable name, for example: `const listItems` instead of `const li`.